src/sass/components/_post-box.scss

```diff
@@ -67,19 +63,17 @@ a.not-underline {
 }
 
 .post-box {
-
   &:hover {
-    text-decoration: none;
-
+    .details {
+      color: black !important;
```

Comment:
What did I say about `!important`? :P Instead of the overriding color, as a first step just check why they are changing. It's because of applying `color: $primary;` in `details`. BTW at line 73 you also have `.details` selector - we don't want to have them split. The solution to this problem is not overriding color with `!important`, but removing selector at line 73.